README.md

```diff
@@ -1,91 +1,90 @@
 # Sern Handler
-<a href="https://www.npmjs.com/package/sern-handler">
-<img src="https://img.shields.io/npm/v/sern_handler?maxAge=3600" alt="NPM version" /></a> <a href="https://www.npmjs.com/package/sern-handler"><img src="https://img.shields.io/npm/dt/sern_handler?maxAge=3600" alt="NPM downloads" /></a> <a href="https://www.npmjs.com/package/sern-handler"><img src="https://img.shields.io/badge/builds-stable" alt="Builds Passing"></a>
 
-Sern can automate and streamline development of your discord bot with new version compatibility and full customization.
+<a href="https://www.npmjs.com/package/@sern/handler">
+<img src="https://img.shields.io/npm/v/@sern/handler?maxAge=3600" alt="NPM version" /></a> <a href="https://www.npmjs.com/package/@sern/handler"><img src="https://img.shields.io/npm/dt/@sern/handler?maxAge=3600" alt="NPM downloads" /></a> <a href="https://www.npmjs.com/package/@sern/handler"><img src="https://img.shields.io/badge/builds-stable" alt="Builds Passing"></a>
 
--   A reincarnation of [this old project](https://github.com/jacoobes/sern_handler)
+A discord.js wrapper for you to automate and streamline your bot development.
+
+- A reincarnation of [this old project](https://github.com/jacoobes/sern_handler)
 
 ## Installation
 
 ```sh
-npm install sern-handler
+npm install @sern/handler
 ```
+
 ```sh
-yarn add sern-handler
+yarn add @sern/handler
 ```
+
 ```sh
-pnpm add sern-handler
+pnpm add @sern/handler
 ```
 
 ## Basic Usage
 
-#### ` index.js `
+#### ` index.js (CommonJS)`
+
 ```js
-import { Client, Intents } from 'discord.js';
-import { Sern } from 'sern-handler';
-import { prefix, token } from '../src/secrets.json';
+const { Client, Intents } = require('discord.js');
```

Comment:
yeah Intents is not anymore in v14, its GatewayIntentBits
```suggestion
const { Client, GatewayIntentBits } = require('discord.js');
```
please look at cheems repo

Reply:
Sounds good, keep repo example v14